README.md

```diff
@@ -11,6 +11,10 @@ control of the schema of the declarative API it offers.
 
 Crossplane is a [Cloud Native Computing Foundation][cncf] project.
 
+## Getting Started
+
+Crossplane's [Getting Started Documentation](https://docs.crossplane.io/v1.13/getting-started/) covers Crossplane installation and cloud provider quickstarts.
```

Comment:
Is there some kind of latest link we can use? Just to remove one more place we'll need to keep the URL fresh as new versions are released.

Other than that, LGTM. Could you wrap at 80 chars to match the rest of the doc though?

Reply:
Use `/latest` instead of the version for an evergreen link

Reply:
updated